Sources/SmokeDynamoDB/AWSDynamoDBCompositePrimaryKeyTable+updateItems.swift

```diff
@@ -487,6 +525,30 @@ public extension AWSDynamoDBCompositePrimaryKeyTable {
         }
     }
     
+    func monomorphicBulkWriteWithFallback<AttributesType, ItemType>(_ entries: [WriteEntry<AttributesType, ItemType>]) async throws {
+        // fall back to singel operation if the write entry exceed the statement length limitation
+        var bulkWriteEntries: [WriteEntry<AttributesType, ItemType>] = []
+        try await entries.concurrentForEach { entry in
+            do {
+                try self.validateEntry(entry: entry)
+                bulkWriteEntries.append(entry)
+            } catch SmokeDynamoDBError.statementLengthExceeded {
+                switch entry {
+                case .update(new: let new, existing: let existing):
+                    try await self.updateItem(newItem: new, existingItem: existing)
+                case .insert(new: let new):
+                    try await self.insertItem(new)
+                case .deleteAtKey(key: let key):
+                    try await self.deleteItem(forKey: key)
+                case .deleteItem(existing: let existing):
+                    try await self.deleteItem(existingItem: existing)
+                }
+            }
+        }
+
+        return try await monomorphicBulkWrite(entries)
```

Comment:
`bulkWriteEntries` instead if `entries` here?

Reply:
yeah, you are right. We should use bulkWriteEntries here. Fixed.